acra/keys/inventory.md

```diff
@@ -41,60 +45,54 @@ Storage keys can be represented by either:
     Using zones complicates unauthorized decryption:
     the attacker not only needs to get the decryption key but to use a correct Zone ID, too.
 
-## Acra components
-
-Each Acra component needs to have its own keystore (located in the `.acrakeys` directory by default)
-which contains a set of keys necessary for correct operation of the component.
-
-Here is an overview of core Acra keys and their locations:
-
-| Purpose | Private key stays on | Public key exchanged with |
-| ------- | -------------------- | ------------------------- |
-| Transport: AcraConnector     | AcraConnector  | → AcraServer or AcraTranslator |
-| Transport: AcraServer        | AcraServer     | → AcraConnector |
-| Transport: AcraTranslator    | AcraTranslator | → AcraConnector |
-| Data encryption (AcraStruct) | AcraServer     | → AcraWriter |
-| Data encryption (AcraStruct) | AcraTranslator | → AcraServer (in Transparent proxy mode) |
-| Authentication storage key (for AcraWebConfig users) | AcraServer | |
-
-**AcraConnector** needs the following keys to establish a [Secure Session connection](/themis/crypto-theory/cryptosystems/secure-session/):
-
-  - AcraConnector’s own transport private key.
-
-  - Other party transport public key.
-
-    You should put the transport public key of AcraServer or AcraTranslator
-    into AcraConnector’s keystore, depending on the connection type.
-
-**AcraServer** works with the following keys:
-
-  - AcraServer’s own transport private key as well as AcraConnector’s transport public key.
+## Key names and locations
 
-    They are necessary for accepting connections from the clients via Secure Session.
+> Note: Read more about handling key names in [Losing your keys]({{< ref "/acra/keys/troubleshooting.md#losing-the-keys" >}}) and [Renaming your keys]({{< ref "/acra/keys/troubleshooting.md#renaming-key-files" >}}).
 
-  - Storage private keys for decrypting AcraStructs generated by AcraWriter.
+Each Acra component has to have its own key storage (by default it is key folder `.acrakeys`) that contains a set of keys necessary for the correct functioning of the components:
 
-  - If AcraServer is running in [Transparent proxy mode](https://docs.cossacklabs.com/pages/documentation-acra/#transparent-proxy-mode),
-    it should possess the storage public keys to be able to encrypt the values from SQL queries into AcraStructs.
+- **AcraConnector** needs to have its own transport private key and other party transport public key to establish a [Secure Session connection]({{< ref "themis/crypto-theory/cryptosystems/secure-session.md" >}}). You should put transport public key of AcraServer or AcraTranslator into AcraConnector's key storage, depending on the connection type.
 
-  - If you’re using *AcraWebConfig* HTTP server to configure AcraServer remotely,
-    the users’ credentials are stored encrypted with authentication storage key
-    and are decrypted by AcraServer upon users’ login.
+- **AcraServer** should have:
```

Comment:
IMO, I would not refer to key names (since they are actually introduced below in correspondent tables), but just explain their high-level purpose, e.g.:
**AcraServer** should have:
1) Keys for accepting connections from clients via SecureSession;
2) Keys for decrypting AcraStructs generated by AcraWriter;
3) Keys for encrypting/decrypting AcraBlocks in transparent mode;
...

Reply:
IMO, the more verbose explanation is better. Because we have different type of keys (asymmetric/symmetric) for same purposes (AcraStruct/AcraBlock encryption). Same for cases with zones.